Dockerfile.simple

```diff
@@ -41,8 +41,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 # IMPORTANT: If the version of Go is updated, the Windows to Linux CI machines
 #            will need updating, to avoid errors. Ping #docker-maintainers on IRC
 #            with a heads-up.
-# IMPORTANT: When updating this please note that stdlib archive/tar pkg is vendored
-ENV GO_VERSION 1.10.3
+ENV GO_VERSION 1.11
 RUN curl -fsSL "https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz" \
```

Comment:
Wondering if we should change the `FROM` to `FROM golang:1.11` or `FROM golang:1.11.0-stretch` instead

Reply:
Just to note, if you change this, I'd have to update the Windows CI script again. It currently copes with the old `ENV GOVERSION=` and `FROM golang:x.xx` only.

Reply:
I was thinking about it, too, but it's out of scope for this PR; will do it later.

Reply:
In fact the change is pretty small so I've added it to this PR. Thanks!